2020-03-Autumn/1_pointer_techniques/README.md

```diff
@@ -105,87 +117,46 @@ Go to [Solution](#s3)   [Top](#top)
 <!-- Don't remove -->
 <a name="s1"/>
 
-### 1. Max Consecutive Ones
+### 1. Remove Duplicates from Sorted Array
 
-Source: [LeetCode](https://leetcode.com/problems/max-consecutive-ones/)
+[LeetCode](https://leetcode.com/problems/remove-duplicates-from-sorted-array/)
 
-#### Brute Force Solution
+#### Optimal solution
 
-The direct solution would go through the array, find the first 1, and 
-start counting all 1s after it until the cursor reaches 0. This solution
-works, however, leads to very bad time complexity (O(N^2)).
+Since the array is already sorted, we can keep two pointers, left and right, where left is the slow runner and right is the fast runner. As long as nums[left] == nums[right], right will continue moving one element forward, skipping the dupplicate. 
```

Comment:
```suggestion
Since the array is already sorted, we can keep two pointers, left and right, where left is the slow runner and right is the fast runner. As long as nums[left] == nums[right], right will continue moving one element forward, skipping the duplicate. 
```